lib/v2/screens/profile_screens/security/components/security_card.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:flutter/material.dart';
```

Comment:
![image](https://user-images.githubusercontent.com/8041901/110877313-3a0cd180-82a7-11eb-97b3-61c558931fc0.png)
![image](https://user-images.githubusercontent.com/8041901/110877324-3da05880-82a7-11eb-9a89-4df9285f0095.png)


security card and settings card look very similar to me, Can we resuse them or do we need both?

Reply:
no need because settigns screen will be removed soon xD